examples/animations/src/main.rs

```diff
@@ -37,7 +38,7 @@ fn app_view() -> impl IntoView {
             .animation(move |_| animation.get())
             .animation(move |a| {
                 a.keyframe(100, |kf| {
-                    kf.style(|s| s.border(5).border_color(Color::PURPLE))
+                    kf.style(|s| s.border(Stroke::new(5.)).border_color(Color::PURPLE))
```

Comment:
Maybe we could make a trait to convert numbers to `StrokeWrap` so that we can still write it the old way.

Reply:
Ah we can. I first started converting them to Stroke::new and after it was too much of a pain I added impls for From<num> for StrokeWrap. I just haven't changed these examples back to just numbers but it would work if I did.

Reply:
@dzhou121 do you want these changed back?

Reply:
Yes please